### DISCUSSION TITLE
does hyprspace support routing?

### DISCUSSION THREAD

**Discussion by jingkang99** [alecbcs/hyprspace#72] [Q&A]
![image](https://user-images.githubusercontent.com/10793075/152494910-76eb55b9-50bc-4aaa-b52b-c43be461f660.png)

The goal is to access internal services, like web app on 10.0.0.79. it isn't effective by add routing rule on 10.1.1.1 and 10.0.0.79
on 10.1.1.1
    route add -net 10.0.0.0 netmask 255.255.255.0 gw 10.1.1.1
    ping 10.0.0.79

tcpdump shows nothing when capturing packets on 10.1.1.95/hs0, no icmp packet coming
tcpdump -i hs0 -n 

of course I can setup hyprspace  on 10.0.0.79.  https://github.com/mudler/edgevpn is ok

btw, where is the vpn Rendezvous point, and how decided?

why?

**Comment by alecbcs** [comment]
@jingkang99 Hyprspace does not yet support defining one of the peer as a router but we will soon. Checkout #62. Once that is merged in I believe you should be able to accomplish that you've drawn above.

**Reply by jingkang99** [reply]
Great. This is great project with potentials. thanks for your effort.
